test_collections/matter/scripts/OTBR/otbr_start.sh

```diff
@@ -49,10 +49,10 @@ print_script_step "Waiting 10 seconds to give the the docker container enough ti
 sleep 10
 
 BR_CHANNEL_HEX=$(printf '%02x' $BR_CHANNEL)
-BR_PANID="5b${BR_VARIANT}"
```

Comment:
I think we should have comments here explaining the choice of values for each of these parameters.

Reply:
I have no idea about the choice of those values in the TH
All I did was to change the "example" values in this file to match with the default values already in the TH config.

The idea is the user change this file anyway for the desired thread configuration they want. Thus the TH as well.

Reply:
We spoke offline and agreed to add some comments to this section.
Added [here](https://github.com/project-chip/certification-tool-backend/pull/133/commits/6701fac193c2ba9f88c0a215080f95df736c8b18)